test/Dialect/Calyx/emit.mlir

```diff
@@ -280,3 +280,54 @@ module attributes {calyx.entrypoint = "main"} {
   } {toplevel}
 }
 
+
+// -----
+
+module attributes {calyx.entrypoint = "main"} {
+  // CHECKL: import "primitives/float/addFN.futil";
```

Comment:
Is this a typo, or does `CHECKL` exist?

Reply:
my bad, I wonder how it even passes the test check..